client/hosting/server-settings/components/cache-card/index.tsx

```diff
@@ -126,57 +82,109 @@ export const CacheCard = ( {
-								onChange={ ( active ) => {
-									recordTracksEvent(
-										active
-											? 'calypso_hosting_configuration_edge_cache_enable'
-											: 'calypso_hosting_configuration_edge_cache_disable',
-										{
-											site_id: siteId,
-										}
-									);
-									setEdgeCache( siteId, active );
-								} }
-								label={ edgeCacheToggleDescription }
-							/>
-							<Hr />
-						</>
-					) }
-				</ToggleContainer>
-				{ getClearCacheContent() }
-			</CardBody>
+				<Button
+					onClick={ handleClearCache }
+					busy={ isClearingCache }
+					disabled={
+						disabled ||
+						isClearingCache ||
+						shouldRateLimitCacheClear ||
+						getEdgeCacheLoading ||
+						isEdgeCacheMutating
+					}
+					className="performance-optimization__button"
+				>
+					<span>{ translate( 'Clear all caches' ) }</span>
+				</Button>
+				<div className="performance-optimization__nb">
+					{ translate( 'Clearing the cache may temporarily make your site less responsive.' ) }
+				</div>
+				{ shouldRateLimitCacheClear && (
+					<div className="performance-optimization__nb">
+						{ translate( 'You cleared the cache recently. Please wait a minute and try again.' ) }
+					</div>
+				) }
+			</div>
+
+			<div className="performance-optimization__hr"></div>
+
+			<div className="performance-optimization__global-edge-cache-block">
+				{ getEdgeCacheInitialLoading ? (
+					<EdgeCacheLoadingPlaceholder />
+				) : (
+					<>
+						<div className="performance-optimization__subtitle">
+							{ translate( 'Global edge cache' ) }
+						</div>
+						<ToggleControl
+							disabled={
+								clearEdgeCacheLoading ||
+								getEdgeCacheLoading ||
+								! isEdgeCacheEligible ||
+								isEdgeCacheMutating
+							}
+							checked={ isEdgeCacheActive && isEdgeCacheEligible }
+							onChange={ ( active ) => {
+								recordTracksEvent(
+									active
+										? 'calypso_hosting_configuration_edge_cache_enable'
+										: 'calypso_hosting_configuration_edge_cache_disable',
+									{
+										site_id: siteId,
+									}
+								);
+								setEdgeCache( siteId, active );
+							} }
+							label={ edgeCacheToggleDescription }
+						/>
+						{ config.isEnabled( 'hosting-server-settings-enhancements' ) && (
+							<Button onClick={ () => {} } className="performance-optimization__button">
+								<span>{ translate( 'Clear edge cache' ) }</span>
+							</Button>
+						) }
+					</>
+				) }
+			</div>
+
+			{ config.isEnabled( 'hosting-server-settings-enhancements' ) && (
+				<div className="performance-optimization__global-object-cache-block">
+					<div className="performance-optimization__subtitle">{ translate( 'Object cache' ) }</div>
+					<HostingCardDescription>
+						{ translate(
+							'Data is cached using Memcached to reduce database lookups. {{a}}Learn more{{/a}}.',
+							{
+								components: {
+									a: <InlineSupportLink supportContext="hosting-clear-cache" showIcon={ false } />,
```

Comment:
Note to self: we need to update this article.

Reply:
Do we have an issue tracked to coordinate updating docs?

Reply:
We have https://github.com/Automattic/dotcom-forge/issues/9126. That's not very specific so far, though. This topic (object cache) might even warrant its own article, in which case I would probably opt to add that as a follow-up item and remove this link when we launch…

Reply:
Thanks. Can we coordinate with @alexapeduzzi and launch doc changes along with the feature?

Reply:
Made the request here: pfA0Z5-70-p2 

Please leave any comments there about things you may want to include, notably which differences between the global cache and object cache you want to highlight.